src/panels/config/integrations/integration-panels/zwave/zwave-migration.ts

```diff
@@ -79,259 +95,275 @@ export class ZwaveMigration extends LitElement {
-                  ? html`
-                      <ha-card class="content" header="Stop Z-Wave Network">
-                        <div class="card-content">
-                          <p>
-                            We need to stop the Z-Wave network to perform the
-                            migration. Home Assistant will not be able to
-                            control Z-Wave devices while the network is stopped.
-                          </p>
-                          ${this._stoppingNetwork
-                            ? html`
-                                <div class="flex-container">
-                                  <ha-circular-progress
-                                    active
-                                  ></ha-circular-progress>
-                                  <div><p>Stopping Z-Wave Network...</p></div>
-                                </div>
-                              `
-                            : ``}
-                        </div>
-                        <div class="card-actions">
-                          <mwc-button @click=${this._stopNetwork}>
-                            Stop Network
-                          </mwc-button>
-                        </div>
-                      </ha-card>
-                    `
-                  : ``}
-                ${this._step === 2
-                  ? html`
-                      <ha-card class="content" header="Set up OZWDaemon">
-                        <div class="card-content">
-                          <p>Now it's time to set up the OZW integration.</p>
-                          ${isComponentLoaded(this.hass, "hassio")
-                            ? html`
-                                <p>
-                                  The OZWDaemon runs in a Home Assistant addon
-                                  that will be setup next. Make sure to check
-                                  the checkbox for the addon.
-                                </p>
-                              `
-                            : html`
-                                <p>
-                                  If you're using Home Assistant Core in Docker
-                                  or a Python venv, see the
-                                  <a
-                                    href="https://github.com/OpenZWave/qt-openzwave/blob/master/README.md"
-                                    target="_blank"
-                                    rel="noreferrer"
+          ${html`
+            ${this._step === 0
+              ? html`
+                  <ha-card class="content" header="Introduction">
+                    <div class="card-content">
+                      <p>
+                        This wizard will walk through the following steps to
+                        migrate from the legacy Z-Wave integration to Z-Wave JS.
+                      </p>
+                      <ol>
+                        <li>Stop the Z-Wave network</li>
+                        <li>
```

Comment:
I was going to suggest to show this conditionally but realized we have no way to query installation type yet, so it's fine like this.

Reply:
Actually, I guess we can make this conditional on if `hassio` integration is loaded.